data/helpers.d/backend

```diff
@@ -51,3 +51,37 @@ ynh_remove_logrotate () {
 		sudo rm "/etc/logrotate.d/$app"
 	fi
 }
+
+# Create a dedicated php-fpm config
+#
+# usage: ynh_add_fpm_config
+ynh_add_fpm_config () {
+	finalphpconf="/etc/php5/fpm/pool.d/$app.conf"
+	ynh_backup_if_checksum_is_different "$finalphpconf" 1
+	sudo cp ../conf/php-fpm.conf "$finalphpconf"
+	ynh_replace_string "__NAMETOCHANGE__" "$app" "$finalphpconf"
```

Comment:
Hum, as you said just before, I think too that nobody change the fpm config file.
And, if I agree with you for the fact that `__NAME__` will be more consistent than `__NAMETOCHANGE__` with the nginx helper.
I think also that will affect all currently app which use php-fpm.


So, maybe it's better to break thoses apps now, before more apps uses this helper...

And, for a default php-fpm file, I think it's a good idea, but not like that. I prefer a default config in the exemple app. And each app copy this file in in own conf directory.
More control, and more aware that you use it.